pkgs/by-name/ha/hanko/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, installShellFiles, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hanko";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "SRv6d";
+    repo = pname;
```

`repo = pname;` is an antipattern as the repo name isn't inherently related to the pname in any way
```suggestion
    repo = "hanko";
```